netdata-installer.sh

```diff
@@ -13,8 +13,14 @@ fi
 LC_ALL=C
 umask 022
 
+DEFCFLAGS=
+case "$((arch))" in
+	i386|i686) DEFCFLAGS="-O3 -msse2 -mfpmath=sse" ;;
```

Comment:
err? you sure i386 cpu has sse2? quick search from [wikipedia](https://en.wikipedia.org/wiki/SSE2) says it came with first pentiums, so i586 should be the minimum.